tools/cldr-code/src/main/java/com/ibm/icu/dev/test/TestFmwk.java

```diff
@@ -1395,6 +1383,8 @@ public void init() {
             invalidCount = 0;
             testCount = 0;
             random = seed == 0 ? null : new Random(seed);
+
+            knownIssues = new UnicodeKnownIssues(allKnownIssues);
```

Comment:
optional: maybe skip this if -prop:logKnownIssue=no ?

Reply:
could, but:
- getBooleanProperty is actually not callable from this line as it's in the owning class
-  it's one object per process which contains a map
- logKnownIssue=yes is the default